### DISCUSSION TITLE
Dynamic delay combined with Mail Merge Batch number

### DISCUSSION THREAD

**Discussion by nuttiplutt** [Extended-Thunder/send-later#452] [Ideas]
Hello, 
Are there any possibilities to make a dynamic send time based on which batch number you have in a mail merge?
To use an example:
I do a mail merge with the extension Mail Merge using a csv file. The first 30 entries can be sent right away or maybe a 5 minute delay. Then the next 30 entries should be sent one hour (or maybe 65 min) later to avoid being marked as spam. Then the next 30 entries 65 minutes after the last 30 entries and so on. 
Now I have to make a new mail merge for every group of 30 entries using the Batch function and setting manually the factual time to send them. 
I would love it if you can make such a function, but of course, it might have to be a collaboration with they who make Mail Merge extension too.

**Comment by jph-sendlater** [comment]
This is a question for the maintainer of the MailMerge extension.

Quick background on why: when you use MailMerge with SendLater, the role of SendLater is just to look at the Drafts folder, and send things according to the assigned schedule. MailMerge is responsible for setting those schedules.

**Comment by jikamens** [comment]
You can put a send date/time or just time column in your CSV and refer to it in the Mail Merge setting when generating the Mail Merge. E.g., if you have a column called "sendat" in your CSV in the format YYYY-MM-DD HH:MM, then you can put {{sendat}} in the Mail Merge dialog for the send time and it'll pull from that column. Or if you want the date to be the same and just have a variable time, you can put a time column in your CSV in HH:MM format and put the date followed by {{time}} in the Mail merge input field. Basically, the send parameters in the Mail Merge dialog will do mail merge just like the message contents do.
